api/rs/slint/mcu.md

```diff
@@ -183,13 +179,13 @@ With a `Platform` in place, you can write the main event loop to drive all the d
 
 You can choose between two options:
 
- * You can implement [`slint::platform::Platform::run_event_loop`]: Use this if you want to start the
-   event loop in a way similar to desktop platforms, using the [`run()`](slint::ComponentHandle::run) function
-   of your component, or use [`slint::run_event_loop()`]. Both of these functions will call your implementation
-   of [`slint::platform::Platform::run_event_loop`].
- * Implement a `loop { ... }` directly in your main function: This is called a super loop architecture and common
-   for programs running in bare metal environments on MCUs. It allows you to initialize you device peripherals
-   and access them without the need to move them into your `Platform` implementation.
+ * Use [`slint::platform::Platform::run_event_loop`] if you want to start the
+   event loop in a way similar to desktop platforms. The
+   [`run()`](slint::ComponentHandle::run) function of your component, and [`slint::run_event_loop()`]
+   will both call [`slint::platform::Platform::run_event_loop`].
+ * Put a `loop { ... }` into your main function: This so called super loop architecture is common
+   for programs running in bare metal environments on MCUs. It allows to initialize device peripherals
+   and access them without the need to move them into the `Platform` implementation.
```

Comment:
I prefer the consistent addressing of the developer with "you". We use it all the time, most recently at the beginning of the section: "You can choose between two options". Why remove it?